setup/helpers/perf_helper.rb

```diff
@@ -307,80 +335,72 @@ def puppet_module_dependencies
     # Hacky, but new versions of puppet require different stuff in Puppetfile,
     # for older versions, remove those lines.
     step "Use external puppet modules for puppet >= 6.0" do
-      puppet_version = on(master, 'puppet --version').stdout
+      staging_prod = "/etc/puppetlabs/code-staging/environments/production"
```

Comment:
Should this use `.freeze`?  Just being cautious since it is being substituted in the `rm -rf` below :)

Reply:
`.freeze` added. :+1: thanks